x-pack/test/security_solution_cypress/cypress/screens/breadcrumbs.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const RULES_TABLE_BREADCRUMB =
+  '[data-test-subj~="breadcrumb"][title="Detection rules (SIEM)"]';
```

Comment:
(Some general thoughts) Breadcrumbs tend to change pretty often. Sometimes their structure changes (the tree), sometimes they are renamed. I think they changed at least twice during the last two releases.

So when SPA navigation is important for testing certain scenarios (such as making sure that the Rules table gets refreshed after some change), I think it might be OK-ish to use breadcrumbs for doing it. Although it would be great to use more stable UI elements (when they exist) or ways for doing SPA navigation. In theory, one way could be using the browser [History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API/Working_with_the_History_API) (`.pushState()` and `.back()` methods) directly from tests. There's an article with a short example: https://glebbahmutov.com/blog/cypress-history-api-example/

When SPA navigation is not required for testing something specific, we should be just opening pages by their URLs.